minerl/env/_multiagent.py

```diff
@@ -530,7 +531,8 @@ def _setup_agent_xmls(self, ep_uid: str) -> List[etree.Element]:
                         <MaxPlayers>{}</MaxPlayers>
                     </HumanInteraction>""".format(self.interact_port, self.max_players))
                 # Update the xml
-                ss = agent_xml_etree.find(".//" + self.ns + 'ServerSection')
+                namespace = '{http://ProjectMalmo.microsoft.com}'
```

Comment:
Why is this no longer `self.ns` or just the `NS` global constant which seems to contain the same value?

Reply:
I don't remember changing this - I think it was pulled from another branch - reverting